otelcolbuilder/.otelcol-builder.yaml

```diff
@@ -76,151 +76,151 @@ receivers:
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/connector/failoverconnector v0.124.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/connector/roundrobinconnector v0.124.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/connector/routingconnector v0.124.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/connector/servicegraphconnector v0.124.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/connector/spanmetricsconnector v0.124.0
 providers:
   - gomod: github.com/SumoLogic/sumologic-otel-collector/pkg/configprovider/globprovider v0.0.0-00010101000000-000000000000
   - gomod: github.com/SumoLogic/sumologic-otel-collector/pkg/configprovider/opampprovider v0.0.0-00010101000000-000000000000
   # Upstream providers
-  - gomod: go.opentelemetry.io/collector/confmap/provider/envprovider v0.118.0
-  - gomod: go.opentelemetry.io/collector/confmap/provider/fileprovider v0.118.0
-  - gomod: go.opentelemetry.io/collector/confmap/provider/httpprovider v0.118.0
+  - gomod: go.opentelemetry.io/collector/confmap/provider/envprovider v0.124.0
+  - gomod: go.opentelemetry.io/collector/confmap/provider/fileprovider v0.124.0
+  - gomod: go.opentelemetry.io/collector/confmap/provider/httpprovider v0.124.0
   - gomod: go.opentelemetry.io/collector/confmap/provider/httpsprovider v1.20.0
-  - gomod: go.opentelemetry.io/collector/confmap/provider/yamlprovider v0.118.0
+  - gomod: go.opentelemetry.io/collector/confmap/provider/yamlprovider v0.124.0
 excludes:
   - github.com/knadh/koanf v1.5.0
 # Replacement paths are relative to the output_path (location of source files)
 replaces:
   # ----------------------------------------------------------------------------
   # Needed for telegrafreceiver
-  - github.com/influxdata/telegraf => github.com/SumoLogic/telegraf v1.32.0-sumo-0
+  - github.com/influxdata/telegraf => github.com/SumoLogic/telegraf v1.34.2-sumo-0
   # This is to make sure we build the with the sumologicextension from the tree.
   # The reason this is needed is that sumologicexporter depends on sumologicextension in its go.mod.
   # We add a dependency in the builder config in this file but only for a particular (placeholder)
   # version which gets then translated into a replace in go.mod file.
   # This does not replace the version that sumologicexporter depends on.
   - github.com/SumoLogic/sumologic-otel-collector/pkg/extension/opampextension => ../../pkg/extension/opampextension
-  - github.com/open-telemetry/opentelemetry-collector-contrib/extension/sumologicextension v0.118.0 => github.com/SumoLogic/opentelemetry-collector-contrib/extension/sumologicextension b8ff5f2de776724cb97a77432189523999ef19e9
+  - github.com/open-telemetry/opentelemetry-collector-contrib/extension/sumologicextension v0.124.0 => github.com/SumoLogic/opentelemetry-collector-contrib/extension/sumologicextension b8ff5f2de776724cb97a77432189523999ef19e9
```

Comment:
Recommend removing this since this change is now included in the release - https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37689

cc: @amdprophet